meta-spec-0002.md

```diff
@@ -77,8 +77,7 @@ The developers of the project do not all reside at a single institution.
 
 Core Projects are **well documented and well tested**.
 They document their APIs, and have inline documentation (most often in
-the form of [numpydoc](https://numpydoc.readthedocs.io/) or
-[napoleon](https://sphinxcontrib-napoleon.readthedocs.io/)).
+the form of the [numpydoc docstring standard](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard).
```

Comment:
The `numpydoc` docstring standard defines the format, which is supported by both the `numpydoc` and `napoleon` sphinx extensions. Note that napoleon also supports the google doc string standard, which is less common (and maybe not encouraged?) in scientific Python packages.